testsuite/MDAnalysisTests/test_persistence.py

```diff
@@ -142,3 +142,26 @@ def test_pickle_unpickle_empty(self):
         pickle_str = cPickle.dumps(ag, protocol=cPickle.HIGHEST_PROTOCOL)
         newag = cPickle.loads(pickle_str)
         assert_equal(len(newag), 0)
+
+def test_pickling_uag():
+    u = mda.Universe(PDB_small)
+
+    uag = u.select_atoms('name C', updating=True)
+
+    pickle_str = cPickle.dumps(uag, protocol=cPickle.HIGHEST_PROTOCOL)
+
+    new_uag = cPickle.loads(pickle_str)
+
+    assert_array_equal(uag.indices, new_uag.indices)
+
```

Comment:
Could you add an explicit test for the unpickling of a UAG from a regular AtomGroup?